.gitignore 2

```diff
@@ -0,0 +1,66 @@
+# This .gitignore file should be placed at the root of your Unity project directory
```

Comment:
Not sure the purpose of this file. Why do we need this as well as the .gitignore file?